code/controllers/subsystems/xenoarch.dm

```diff
@@ -46,6 +47,9 @@ SUBSYSTEM_DEF(xenoarch)
 					continue
 				if(T in processed_turfs)
 					continue
+				var/area/A = get_area(T)
+				if(!A.allow_xenoarchaeology_finds)
```

Comment:
i dunno if i'm a big fan of this? i feel like maybe we should make some more fitting archaeology finds like plant and animal remains, etc.

Reply:
would be neat if we could configure find types per map and just remove `/decl/archaeological_find/material`, `/decl/archaeological_find/material/exotic`, `/decl/archaeological_find/parts`, `/decl/archaeological_find/gun`, `/decl/archaeological_find/laser`, `/decl/archaeological_find/mask`, `/decl/archaeological_find/beacon`, `/decl/archaeological_find/remains/robot`... oh, and make `/obj/item/chems/glass/replenishing` have its list of chems be per-map too, probably